.eslintrc.js

```diff
@@ -124,6 +124,8 @@ module.exports = {
       extends: [...baseExtends, 'plugin:@typescript-eslint/recommended'],
       rules: {
         ...baseRules,
+        'no-shadow': 'off',
+        '@typescript-eslint/no-shadow': 'error',
```

Comment:
This config is for fixing a bug where eslint throws an error on enums. For more info check the following [link](https://github.com/typescript-eslint/typescript-eslint/issues/2466)